src/css/styles.scss

```diff
@@ -259,13 +259,10 @@ ul {
 }
 
 .friend-button {
-  background-color: $header_color;
-  border-radius: 50%;
-  border: none;
-  color: #ffffff;
-  height: 65px;
-  width: 65px;
+  @include submitButtons;
+  margin: 0px 3%;
+  padding: 3%;
   text-align: center;
-  font-size: $font_size_small;
+  font-size: $font_size_xsmall;
   cursor: pointer;
```

Comment:
The @include already has the pointer property so shouldn't need to add again here